receiver/mysqlreceiver/metadata.yaml

```diff
@@ -313,6 +313,51 @@ metrics:
       monotonic: true
       aggregation_temporality: cumulative
     attributes: [io_waits_operations, table_name, schema]
+  mysql.table.rows:
+    enabled: false
+    description: The number of rows for a given table.
+    unit: 1
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation_temporality: cumulative
+    attributes: [table_name, schema]
+  mysql.table.average_row_length:
+    enabled: false
+    description: The average row length in bytes for a given table.
+    unit: By
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation_temporality: cumulative
+    attributes: [table_name, schema]
+  mysql.table.total_length:
+    enabled: false
+    description: The total length (sum of data_length and index_length) in bytes for a given table.
+    unit: By
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation_temporality: cumulative
+    attributes: [table_name, schema]
+  mysql.table.data_length:
+    enabled: false
+    description: The data length in bytes for a given table.
+    unit: By
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation_temporality: cumulative
+    attributes: [table_name, schema]
+  mysql.table.index_length:
+    enabled: false
+    description: The index length in bytes for a given table.
+    unit: By
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation_temporality: cumulative
+    attributes: [table_name, schema]
```

Comment:
Can we just model this as one metric with two data points and an attribute which is either data or index? This is how OTel's data model is intended to work, rather than just returning sums as separate metrics.

Reply:
Yes, that makes sense. I'll remove the `total_length` metric.

Reply:
Also, typically can we name this metric `mysql.table.size`?

Reply:
I misunderstood your last comment. I'll have that changed. Thank you for the review!

Reply:
I still see two separate metrics

Reply:
@djaglowski Please check now. Thank you

Reply:
@djaglowski Checking in - please let me now if there needs additional changes